frmts/jp2kak/jp2kakdataset.cpp

```diff
@@ -2869,6 +2942,24 @@ void GDALRegister_JP2KAK()
 "   <Option name='FLUSH' type='boolean' />"
 "   <Option name='NBITS' type='int' description="
 "'BITS (precision) for sub-byte files (1-7), sub-uint16 (9-15)'/>"
+"   <Option name='RATE' type='string' description='One or more bit-rates, expressed in terms of the ratio between"
+"           the total number of compressed bits (including headers) and the "
+"           product of the largest horizontal and  vertical image component "
+"           dimensions.  A dash, -, may be used in place of the first "
+"           bit-rate in the list to indicate that the final quality layer "
+"           should include all compressed bits."
+"           If Clayers is not used, the number of layers is "
+"           set to the number of rates specified here. If Clayers is used "
+"           to specify an actual number of quality layers, one of the "
+"           following must be true: 1) the number of rates specified here is "
+"           identical to the specified number of layers; or 2) one or two "
+"           rates are specified using this argument.  When two rates are "
+"           specified, the number of layers must be 2 or more and intervening "
+"           layers will be assigned roughly logarithmically spaced bit-rates. "
+"           When only one rate is specified, an internal heuristic determines "
+"           a lower bound and logarithmically spaces the layer rates over the "
+"           range. The rates have to be in ASC order.'/>"
+"   <Option name='Creversible' type='string'/>"
```

Comment:
```suggestion
"   <Option name='Creversible' type='boolean'/>"
```

Reply:
Fix in b9892f387184d438931da1ea1e69899388b78db3